src/components/CommandBar/CommandBarTrigger/CommandBarTrigger.module.scss

```diff
@@ -2,25 +2,60 @@
 
 $width: calc(11 * var(--spacing-mega));
 
+.leftSection {
+  display: flex;
+  align-items: center;
+
+  > svg {
+    fill: var(--color-text-faded);
+    opacity: var(--opacity-50);
+    width: calc(1.4 * var(--spacing-medium));
+  }
+
+  .placeholder {
+    margin-inline-start: var(--spacing-small);
+  }
+}
+
 .container {
+  position: relative;
+  transition: box-shadow var(--transition-fast) ease,
+    top var(--transition-fast) ease;
+
+  background: var(--color-background-default);
+  @media (prefers-color-scheme: dark) {
+    background-color: var(--color-background-alternative-deep);
+    box-shadow: none;
+    &:hover {
+      box-shadow: none;
+    }
+  }
+  [data-theme="dark"] & {
+    background-color: var(--color-background-alternative-faded);
+    box-shadow: none;
+    &:hover {
+      box-shadow: none;
+    }
+  }
```

Comment:
This is not an ideal conditional. We probably need new token `surface`.
We can do this in a follow up

On dark mode it will be a `lighter dark`
On light mode it will be just white.

Currently our popup and modal is using `white shadow` which is weird

#741